src/record/sdk/record.cpp

```diff
@@ -373,7 +380,10 @@ k4a_result_t k4a_record_add_tag(const k4a_record_t recording_handle, const char
         return K4A_RESULT_FAILED;
     }
 
-    add_tag(context, name, value);
+    if (NULL == add_tag(context, name, value))
```

Comment:
Out of scope for this review, but do we need to care about the return value from any of the other add_tag calls?